src/extension/features/budget/subtract-upcoming-from-available/index.js

```diff
@@ -3,6 +3,7 @@ import { isCurrentRouteBudgetPage } from 'toolkit/extension/utils/ynab';
 import { getEmberView } from 'toolkit/extension/utils/ember';
 import { formatCurrency } from 'toolkit/extension/utils/currency';
 import { addToolkitEmberHook, l10n } from 'toolkit/extension/utils/toolkit';
+import { getTotalSavings } from 'toolkit/extension/features/budget/show-available-after-savings/index';
 
 export class SubtractUpcomingFromAvailable extends Feature {
```

Comment:
if you can add a `destroy()` method that cleans up any elements on the screen once someone turns this feature off , that'd be great!